zcash_client_sqlite/src/wallet/sapling.rs

```diff
@@ -186,6 +198,15 @@ pub(crate) fn select_spendable_sapling_notes(
     anchor_height: BlockHeight,
     exclude: &[ReceivedNoteId],
 ) -> Result<Vec<ReceivedSaplingNote<ReceivedNoteId>>, SqliteClientError> {
```

Comment:
The partially duplicated code between this and `get_spendable_sapling_notes` seems like a maintenance hazard. I see that the SQL statement is different and has an extra `:target_value` parameter; can the rest of these methods be abstracted out?

Reply:
The fact that we have two separate such methods is a historical artifact. We should clean it up some day.